nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/bigquery/BadTypeNameException.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.nifi.processors.gcp.bigquery;
+
+/**
+ * Exception thrown when a given type can't be transformed into a valid BigQuery type.
+ * @author nicolas-delsaux
+ *
+ */
+public class BadTypeNameException extends RuntimeException {

```

Comment:
It is a one time used class, so I think it is not needed to provide all the overloaded constructors.

Furthermore, the exception class might be moved to BigQueryUtils as an inner class, because it is only instantiated there and the exception is not caught outside of the class.